flink-examples-gcp/sql/hbaseBT.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE Words (
+  word VARCHAR(32),
+  ts AS PROCTIME()
+) WITH (
+  'connector' = 'datagen',
+  'fields.word.var-len' = 'true',
+  'rows-per-second' = '10'
+);
+
+CREATE TABLE BigtableSink (
```

Comment:
Was checking this out of curiosity.

Could we name this HBaseSink or BigtableHBaseSink? Since I plan to add TableAPI for the in-house BTsink